addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/SpringActuatorScanRule.java

```diff
@@ -0,0 +1,202 @@
+            "Content-Type: application/vnd.spring-boot.actuator.v2+json;charset=UTF-8";
+    private static final String JSON_CONTENT_TYPE = "Content-Type: application/json";
+    private static final Pattern CONTENT_TYPE =
+            Pattern.compile(
+                    "application\\/vnd\\.spring-boot\\.actuator\\.v[0-9]\\+json|application\\/json",
+                    Pattern.MULTILINE);
+    private static final Pattern JSON_PAYLOAD = Pattern.compile("\\{.*\\:.*\\}", Pattern.MULTILINE);
+
+    @Override
+    public boolean targets(TechSet technologies) {
+        // TODO change this to Spring, but need to extend the Tech in Core ZAP first
+        return technologies.includes(Tech.JAVA);
+    }
+
+    @Override
+    public int getId() {
+        return PLUGIN_ID;
+    }
+
+    @Override
+    public String getName() {
+        return Constant.messages.getString(MESSAGE_PREFIX + "name");
+    }
+
+    @Override
+    public String getDescription() {
+        return Constant.messages.getString(MESSAGE_PREFIX + "desc");
+    }
+
+    @Override
+    public String getReference() {
+        return Constant.messages.getString(MESSAGE_PREFIX + "refs");
+    }
+
+    @Override
+    public int getCategory() {
+        return Category.INFO_GATHER;
+    }
+
+    @Override
+    public String getSolution() {
+        return Constant.messages.getString(MESSAGE_PREFIX + "soln");
+    }
+
+    @Override
+    public int getRisk() {
+        return Alert.RISK_MEDIUM;
+    }
+
+    @Override
+    public int getWascId() {
+        return 13;
+    }
+
+    private String getAlertName() {
+        return getName();
+    }
+
+    private static String getOtherInfo(String type) {
+        return Constant.messages.getString(MESSAGE_PREFIX + "otherinfo");
+    }
+
+    @Override
+    public void scan() {
+        String[] endpointList = {"actuator/health"};
+        String[] encodingTypes = {null, "application/json"};
+        for (String endpoint : endpointList) {
+            if (isStop()) {
+                LOG.debug("Scan rule {} stopping.", getName());
+                return;
+            }
+            for (String encodingType : encodingTypes) {
+                HttpMessage testMsg = sendActuatorRequest(encodingType, endpoint);
+                if (testMsg == null) {
+                    continue;
+                }
+                String contentType = testMsg.getResponseHeader().getNormalisedContentTypeValue();
+                if (isPage200(testMsg)) {
+                    String responseBody = testMsg.getResponseBody().toString();
+                    // If all the content checks matched then confidence is high
+                    boolean matches =
+                            CONTENT_TYPE.matcher(contentType).find()
+                                    && JSON_PAYLOAD.matcher(responseBody).find();
+                    if (matches) {
+                        raiseAlert(testMsg, Alert.CONFIDENCE_MEDIUM, getRisk(), endpoint);
```

Comment:
Raise the alert with `CONFIDENCE_HIGH` as the comment states?

Reply:
And remove the comment.